### PR TITLE
Bug 1382006 - change blocklist opt-out information link to SUMO

### LINKED ISSUE #873: Firefox Privacy Page links to incorrect opt-out information
See: https://bugzilla.mozilla.org/show_bug.cgi?id=1382006

### PR DESCRIPTION
**Pull request by MikkCZ**
fix #873

### CODE REVIEW

**Comment by MikkCZ** [comment]
@alexgibson @kyoshino May I ask you for review?

File: firefox-cliqz_privacy_notice/de.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Firefox mit Cliqz stellt automatisch eine Verbindung mit Cliqz her, um Suchergeb
...
 {: #auto-updates }
 
-* __Technische Daten für Add-on-Blockierliste__: Firefox für Desktop und Android stellen von Zeit zu Zeit eine Verbindung mit Mozilla her, um Sie und Andere vor schädlichen Add-ons zu schützen.  Ihre Firefox-Version und Sprache, das Gerätebetriebssystem und die Liste installierter Add-ons werden benötigt, um die Add-on-Blockierliste anzuwenden und zu aktualisieren. [Weitere Informationen](https://blog.mozilla.org/addons/how-to-opt-out-of-add-on-metadata-updates/).
+* __Technische Daten für Add-on-Blockierliste__: Firefox für Desktop und Android stellen von Zeit zu Zeit eine Verbindung mit Mozilla her, um Sie und Andere vor schädlichen Add-ons zu schützen.  Ihre Firefox-Version und Sprache, das Gerätebetriebssystem und die Liste installierter Add-ons werden benötigt, um die Add-on-Blockierliste anzuwenden und zu aktualisieren. [Weitere Informationen](https://support.mozilla.org/kb/how-stop-firefox-making-automatic-connections).
```
Comment (peiying2) [inline firefox-cliqz_privacy_notice/de.md]: For German, it should be this: https://support.mozilla.org/de/kb/Firefox-baut-unaufgeforderte-Verbindungen-auf.

**Comment by peiying2** [comment]
Good catch on the URLs.  I will leave the URLs as you suggested and hopefully the redirect works for all locales. Not all the URL path contain English names.

File: firefox-cliqz_privacy_notice/de.md (hunk 1 above)
Comment (kyoshino) [reply firefox-cliqz_privacy_notice/de.md]: The language neutral link will be redirected there for German:
https://support.mozilla.org/kb/how-stop-firefox-making-automatic-connections
-> https://support.mozilla.org/de/kb/how-stop-firefox-making-automatic-connections
-> https://support.mozilla.org/de/kb/Firefox-baut-unaufgeforderte-Verbindungen-auf 😺

**Pull request merged**